src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -935,26 +938,27 @@ public ChatRoomMemberRole getRoleForMucJid(String mucJid)
      * {@inheritDoc}
      */
     @Override
-    public void onSessionAccept( JingleSession peerJingleSession,
+    public XMPPError onSessionAccept( JingleSession peerJingleSession,
```

Comment:
Highly subjective and highly insignificant, but I think I would have liked better throwing an `XMPPException` here. That would also lead to less changes in the code.